lib/manager/components/version/FeedVersionTabs.js

```diff
@@ -132,6 +146,16 @@ export default class FeedVersionTabs extends Component<Props> {
                 <p style={{marginBottom: '0px'}}>{this.messages('daysActive')}</p>
               </Col>
             </Row>
+            <Row style={{marginTop: '40px'}}>
```

Comment:
@landonreed what do you think of the below:
- With the proposed positioning of the selector for the compared version, I still think the best place for the validity date comparator is below the existing valid date heading.
- One alternate layout could be to move the compared version selector inside the top portion of the summary tab, in which case it makes sense to keep the date comparison chart at this location.

Reply:
My main issue with placing the chart in the valid date heading: when there's a GTFS+ blocking issue, the warning text gets displaced by the chart (see screenshot). Also, I think it makes some sense to place the chart with the other "date oriented" charts (i.e., the histograms showing trips/service hours).

![image](https://user-images.githubusercontent.com/2370911/103556787-47a97380-4e80-11eb-80e6-153e99e01e1e.png)

Given that the compared version selector does affect elements across the entire page, maybe it makes sense to move it next to the green "Create new version button". There is some other work on the roadmap to move the "Edit feed" to its own box below the page selector (Version summary, validation issues, etc.), so that could work well.